compiler_opt/es/blackbox_optimizers.py

```diff
@@ -507,7 +516,9 @@ class ProjectedGradientOptimizer(object):
    i.e. the input variables will not be mutated.
   """
 
-  def __init__(self, function_object, projection_operator, pgd_params, x_init):
+  def __init__(self, function_object: QuadraticModel,
+               projection_operator: Callable, pgd_params: Dict[str, Any],
```

Comment:
Callable from what to what?

Reply:
The function details have been added now